app/controllers/application.js

```diff
@@ -40,7 +40,6 @@ export default Ember.Controller.extend(cancelOrderMixin, {
     return ["browse", "package_category"].indexOf(this.get("currentPath")) >= 0;
   }),
 
-  displayCart: false,
```

Comment:
controllers/item.js has this action
```
    requestItem(item) {
      this.get("cart").add(item);
      Ember.run.later(
        this,
        function() {
          this.get("application").send("displayCart");
        },
        50
      );
    },
```

I hope removing this wont affect this action @patrixr